src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -85,7 +88,9 @@ public IEnumerable<Project> GetSolutionProjects()
             foreach (var hierarchy in EnumerateProjects(solution))
             {
                 Project Project = GetProject(hierarchy);
-                if (Project != null && !ProjectToLanguageMapper.GetLanguageForProject(Project).Equals(Language.Unknown))
+
+                if (Project != null && 
+                    !projectToLanguageMapper.GetAllBindingLanguagesForProject(Project).Any(x=> x.Equals(Language.Unknown)))
```

Comment:
Updated to support the new multiple signature, instead of the old one that was marked Obsolete.

Reply:
The logic looks suspect here too - the `Any` implies there could be multiple languages, one of which is `Unknown`. In practice that shouldn't happen - I guess we're assuming that if `Unknown` occurs it is the only entry.

 It feels as if there is a missing `HasSupportedLanguage` method.